### PR TITLE
Add netherwart recipe

### PR DESCRIPTION
**Pull request by nktruong**
Inspired by honeycomb file

### CODE REVIEW

**Review by epolixa — CHANGES_REQUESTED** [review]
one minor fix:
- the item names should be "minecraft:nether_wart_block" and "minecraft:nether_wart"

couple of nitpicks, not necessary but will fix later if you don't:
- crafting recipes should follow a convention like 'crafted_item_from_item', such as "nether_wart_from_nether_wart_block"
- crafting recipes should be accompanied by a related recipe advancement that controls unlocking the recipe in the player's knowledge book (see advancements/recipes)

File: data/shroomhearth/recipes/netherwart.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "type": "crafting_shapeless",
+    "ingredients": [
+      {
+        "item": "minecraft:netherwart_block"
```
Comment (nktruong) [inline data/shroomhearth/recipes/netherwart.json:5]: change to nether_wart_block

**Comment by nktruong** [comment]
- Updated file to "nether_wart_from_nether_wart_block"

**Comment by nktruong** [comment]
> crafting recipes should be accompanied by a related recipe advancement that controls unlocking the recipe in the player's knowledge book (see advancements/recipes)

- completed 3 years ago

**Review by epolixa — APPROVED** [review]

**Pull request merged**